haskell/test/Test/Ligo/BaseDAO/Management.hs

```diff
@@ -82,70 +80,7 @@ test_BaseDAO_Management =
     , nettestScenarioCaps "invalidates pending owner if new owner is current admin" $ do
         now <- getNow
         invalidatePendingOwner withOriginated (initialStorage now)
-
-    , nettestScenarioCaps "Respects migration state" $ do
-        now <- getNow
-        respectMigratedState withOriginated (initialStorage now)
     ]
-  , testGroup "Accept Ownership"
-      [ nettestScenarioCaps "authenticates the sender" $ do
-          now <- getNow
-          authenticateSender withOriginated (initialStorage now)
-
-       , nettestScenarioCaps "changes the administrator to pending owner" $ do
-          now <- getNow
-          changeToPendingAdmin withOriginated (initialStorage now)
-
-       , nettestScenarioCaps "throws error when there is no pending owner" $ do
-          now <- getNow
-          noPendingAdmin withOriginated (initialStorage now)
-
-       , nettestScenarioCaps "throws error when called by current admin, when pending owner is not the same" $ do
-          now <- getNow
-          pendingOwnerNotTheSame withOriginated (initialStorage now)
-
-      , nettestScenarioCaps "Respects migration state" $ do
```

Comment:
Resolved

Reply:
What about the typescript files?

Reply:
Yes you are right, the typescript files should be updated as well as the parameter has changed.

We should have a check for this, I'll open a new issue to add that :ok_hand:

Reply:
Found a small issue with our Makefile, so I pushed a fix with the changes as well, check it out.

Also opened #184 to avoid this from happening again